src/util/languageUtils.test.js

```diff
@@ -60,6 +61,13 @@ describe('getBrowserLang', () => {
 		const lang = getBrowserLang(request, supportedLangs);
 		expect(lang).toBe(false);
 	});
+	it('returns en-US instead of en-AU for "en" if en-US is preferred', () => {
```

Comment:
it does ([docs](https://www.npmjs.com/package/accepts#languagelanguages)) - you were getting en-AU because `Accepts` was seeing that your first language preference was `en`, which corresponds to "any variant of English", and the first match for "any variant of English" in our app's `supportedLangs` array was `en-AU`. I've now made sure that our `supportedLangs` array is sorted so that `en-US` is first, which means that `en` will always go to `en-US`.

Technically, having `en` as your browser's top language pref makes having any other variant of English in the language preference list redundant. `en` should be specified _after_ all other specific variants have been tried.

I've now added a test to confirm that non-first-preferred languages can still be matched.